workflow-job

```diff
@@ -466,6 +476,26 @@ flows["${sha1}"] = {
             dockeringGuiceCassandraLdap.success()
         }
 
+        stage "Deploying Guice Server with JPA then provision data ${buildId}"
+        node {
+	        sh "cd dockerfiles/run/guice/provisioned; cp Dockerfile NewDockerfile; sed -i -- 's,linagora/james-jpa-guice,${images.jamesJPA},g' NewDockerfile"
+
+	        sh "cd dockerfiles/run/guice/provisioned; docker build --tag=${images.jamesJpaSampleForTesting} -f NewDockerfile ."
+
+	        dockeringGuiceJPASampleTesting.success()
+
+	        sh "docker run --detach=true --name=${containers.jamesJpaSampleForTesting} --hostname ${hostname} --expose=143 --publish-all=true ${images.jamesJpaSampleForTesting}"
```

Comment:
I think the `expose=143` is redondant with the `--publish-all`, if I understood corectly this:
`
First, you can supply -P or --publish-all=true|false to docker run which is a blanket operation that identifies every port with an EXPOSE line in the image’s Dockerfile or --expose <port> commandline flag and maps it to a host port somewhere within an ephemeral port range. The docker port command then needs to be used to inspect created mapping. The ephemeral port range is configured by /proc/sys/net/ipv4/ip_local_port_range kernel parameter, typically ranging from 32768 to 61000.
`
I can see the 143 port listed on jpa/Dockerfile